FlowCrypt/Common/Services/Logger.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-func log(_ message: String, error: Error?, res: Any?, start: DispatchTime) {
+func log(_ message: String, error: Error? = nil, res: Any? = nil, start: DispatchTime = .now()) {
```

Comment:
As noted elsewhere, the start should be passed explicitly to know how long it took, and therefore there it does not need a default.